Makefile

```diff
@@ -458,11 +458,16 @@ bin/helm:
 	mv $(TMP)/linux-amd64/helm bin/helm
 
 .PHONY: values.yaml
-values.yaml:
+values.yaml: values.yaml/calico values.yaml/tigera-operator
+values.yaml/%:
+ifndef RELEASE_STREAM
```

Comment:
I don't think we need `$RELEASE_STREAM` now that we're on branch-per-version. Is this just copy-pasta?